src/docs/development/ui/interactive.md

```diff
@@ -256,9 +256,12 @@ it. In the same location, create the stateful widget:
          ],
        ),
      );
-@@ -117,2 +108,2 @@
-     );
-   }
+@@ -114,3 +105,5 @@
+           ),
+         ),
+       ],
++    );
++  }
```

Comment:
Hey @dnfield, just checking: you ran the refresh script under Linux to get this result, right? (I've resumed investigation of this variance in behavior in macOS vs Linux.)

Good idea to commit the Linux refresh results so that you could get tests to pass.

Reply:
Correct.  I didn't get to dig into the platform differences much - I was a bit surprised by that, although I wonder if it's differences between `diff` on Linux and macOS at this point.

Reply:
Yeah, it is due to `diff` version difference, see #3916